src/main/java/com/library/service/AuthorService.java

```diff
@@ -0,0 +1,16 @@
+package com.library.service;
+
+
+import com.library.model.dto.AuthorDto;
+
+
+public interface AuthorService {
+
+    AuthorDto findById(Long id);
```

Comment:
As we discussed earlier.
In CRUD-controller it is needed to have getAll endpoint.

Reply:
I added already this method. In new commit you will see it.